modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/types.ts

```diff
@@ -85,3 +91,13 @@ export interface ObjectRelationshipEdgeData {
 	targetY: number;
 	type: string;
 }
+
+export type nonRelationshipObjectFieldsInfo = {
+	label: LocalizedValue<string>;
+	name: string;
+};
+
+export type RightSidebarType =
+	| 'objectDefinitionDetails'
+	| 'objectRelationshipDetails'
+	| 'empty';
```

Comment:
Hey @igor-franca can you please sort this type in alphabetical order?

Reply:
Done!